paymentsheet/src/main/java/com/stripe/android/link/LinkControllerViewModel.kt

```diff
@@ -115,7 +115,8 @@ internal class LinkControllerViewModel @Inject constructor(
             },
             getLaunchMode = { _, state ->
                 LinkLaunchMode.PaymentMethodSelection(
-                    selectedPayment = state.selectedPaymentMethod?.details
+                    selectedPayment = state.selectedPaymentMethod?.details,
+                    shareAfterCreatingInPassthroughMode = false,
```

Comment:
👀 `shareAfterCreatingInPassthroughMode` is intentionally awkward because it's a stopgap for more fixes and refactoring later.